src/wrappers/uavcan/si/sample/acceleration/Scalar_1_0.hpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * This software is distributed under the terms of the MIT License.
+ * Copyright (c) 2020 LXRobotics.
+ * Author: Bernhard Mayer <bernard@generationmake.de>, Alexander Entinger <alexander.entinger@lxrobotics.com>
+ * Contributors: https://github.com/107-systems/107-Arduino-UAVCAN/graphs/contributors.
+ */
+
+#ifndef ARDUINO_UAVCAN_TYPES_UAVCAN_SI_SAMPLE_ACCELERATION_SCALAR_1_0_HPP_
+#define ARDUINO_UAVCAN_TYPES_UAVCAN_SI_SAMPLE_ACCELERATION_SCALAR_1_0_HPP_
+
+/**************************************************************************************
+ * INCLUDE
+ **************************************************************************************/
+
+#include <libcanard/canard.h>
+
+#include <types/uavcan/si/sample/acceleration/Scalar_1_0.h>
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
+namespace uavcan {
+namespace si {
+namespace sample {
+namespace acceleration {
+namespace scalar {
```

Comment:
Looking through the other files I've seen that I'm only using the folders towards the type as namespces, not the type itself. E.g. a wrapper stored in `uavcan/node/Heartbeat_1_0.hpp` contains the following namespaces
```C++
namespace uavcan {
namespace node {
```
Please change this in both `Scalar_1_0.hpp` and `Vector3_1_0` - unless I'm missing something and there are stringent reasons for putting it in those namespaces.

Reply:
Thanks for the review! This is a very minor thing, but it also appears to me that your comments on the closing brackets of the namespaces are reversed to mine. Or rather, mine are reversed. Which one do you prefer, leaving them in the opening order so that it's easier to see if one is missing or having them in the correct closing order so that the comments match the actual closed namespace?

Reply:
I prefer

>having them in the correct closing order so that the comments match the actual closed namespace?

but I know, that I sometimes got that wrong myself :blush: